staging/src/k8s.io/api/autoscaling/v1/types.go

```diff
@@ -22,6 +22,24 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// MetricSpecsAnnotation is the annotation which holds non-CPU-utilization HPA metric
+	// specs when converting the `Metrics` field from autoscaling/v2beta1 or autoscaling/v2beta2
+	MetricSpecsAnnotation = "autoscaling.alpha.kubernetes.io/metrics"
```

Comment:
why are these being moved externally? I would not expect anyone to be using these other than conversion

Reply:
OK, i've got your point, thanks.